index.js

```diff
@@ -1,3 +1,4 @@
+const { DateTime } = luxon; // eslint-disable-line
```

Comment:
**Kindly fix this issue:**
I can't change between pages. If I click on "Add New" doesn't do anything. And the same thing for the others. 
![Screen Shot 2021-08-12 at 20 16 26](https://user-images.githubusercontent.com/22258274/129291328-b0e0b50f-2548-4304-8856-a1b851fb2aa8.png)